build.gradle

```diff
@@ -82,6 +82,7 @@ task codeCoverageReport(type: JacocoReport) {
 
   reports {
     xml.enabled true
+    xml.setDestination(file("${buildDir}/reports/jacoco/report.xml"))
```

Comment:
code coverage does it refers to Jacoco reports?

Reply:
yes, it does